localrc-template

```diff
@@ -24,6 +24,7 @@ export EC_POLICY= # e.g. silver
 export SERVERS_PER_PORT=0  # 1 port per dev in ring; N obj-servers per port
 export CONTAINER_AUTO_SHARD=true  # or false
 export OBJECT_SYNC_METHOD=rsync  # or ssync
+export USE_PYTHON3=false  # or true
```

Comment:
maybe.... true :grimacing:

Reply:
I thought about it... wanted to get more func tests passing on py3, and maybe even probe. Not as useful as a dev/review platform otherwise, I feel like.

But hey, if you want to play on hard mode... I can get behind that. I feel like committing to py3-by-default was part of what got me so motivated to get unit tests passing.